x-pack/plugins/cloud_security_posture/server/fleet_integration/fleet_integration.ts

```diff
@@ -85,7 +86,7 @@ export const onPackagePolicyPostCreateCallback = async (
  * Callback to handle deletion of PackagePolicies in Fleet
  */
 export const removeCspRulesInstancesCallback = async (
-  deletedPackagePolicy: DeletePackagePoliciesResponse[number],
+  deletedPackagePolicy: DeepReadonly<DeletePackagePoliciesResponse[number]>,
```

Comment:
This fixes a type issue that was revealed when we added an array field to the `PackagePolicyPackage` type that gets used in the `DeletePackagePoliciesResponse` type.

I think updating this type to expect a `DeepReadOnly` is actually just correct based on the usage of this callback.